packages/runtime/src/builder/serialization/control-serialization.ts

```diff
@@ -1044,7 +1044,18 @@ export function deserializeControls(
 ): Record<string, DeserializedControl> {
   return Object.entries(serializedControls).reduce(
     (deserializedControls, [key, serializedControl]) => {
-      return { ...deserializedControls, [key]: deserializeControl(serializedControl) }
+      try {
+        const deserializedControl = deserializeControl(serializedControl)
+        return { ...deserializedControls, [key]: deserializedControl }
+      } catch (err: unknown) {
```

Comment:
I'd write a unit test for this.

Reply:
https://github.com/makeswift/makeswift/pull/818/files#diff-5cbd6458c52f743b6f82f50f72b22da93215b86db7e90b3c9dc89a61770e8e98